.github/CODEOWNERS

```diff
@@ -0,0 +1,25 @@
+# default option unless later match takes precedence
+* @pietfried @hikinggrass
+
+# .github
+/.github/ @pietfried @corneliusclaussen
```

Comment:
```suggestion
# .github
/.github/ @pietfried @corneliusclaussen
/.github/workflows/ @andistorm 
/.ci/ @andistorm
```

I would like to be added as CI codeowner not sure how the codeowner are resolved in case of defining `/.github/` and `/.github/workflows/`